apps/storybook/package.json

```diff
@@ -6,7 +6,7 @@
     "node": "16.x"
   },
   "scripts": {
-    "start": "start-storybook --quiet -c config -p 3001",
+    "start": "start-storybook --quiet --no-open -c config -p 3001",
```

Comment:
?

Reply:
You should add STORYBOOK_NO_OPEN=true to your .env file

Reply:
Sorry for that, I forgot to remove before commit.